src/Model/Widget/Text.php

```diff
@@ -26,6 +27,22 @@ public static function fromString(string $string): self
         }, explode("\n", $string)));
     }
 
+    public static function parse(string $string): self
+    {
+        $placeholder = sprintf('{{new_line_%s}}', microtime(true));
```

Comment:
`uniqid` would be safer than `microtime`

Reply:
but can also be a static string: `%%__UNLIKELY_STRING_HERE__%%`

Reply:
Yes, I thought about a static string, but I was concerned about conflicts. Maybe we can use something like: `{{php-tui-new-line-placeholder}}`

Reply:
or just `1162128a-269a-4c09-88be-effc70a62a3a` :D

Reply:
That works haha